front-end/src/components/ContactsTable.tsx

```diff
@@ -9,6 +10,30 @@ export default function ContactsTable({
     return null;
   }
 
+  async function sendMessage(slackUserId: string | null) {
+    if (!slackUserId) {
+      return;
+    }
+    
+    try {
+      await fetch(
+        `${baseUrl}/send-slack-message`,
+        {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json'
+          },
+          body: JSON.stringify({
+            integration: 'slack',
+            slackUserId: slackUserId
+          })
+        }
+      );
+    } catch (err) {
+      console.error(err);
```

Comment:
Can you display the error in the row (maybe just after the button) so that's it not hidden from user